code/game/objects/items/AI_modules/full_lawsets.dm

```diff
@@ -69,6 +69,17 @@
 /obj/item/ai_module/core/full/asimovpp
```

Comment:
```suggestion
/obj/item/ai_module/core/full/asimov/pp
```

Reply:
/obj/item/ai_module/core/full/asimovpp is the referenced object in the code before I made adjustments.